### DISCUSSION TITLE
Anyone know of a plugin where you can create item placeholders for items you don't have yet?

### DISCUSSION THREAD

**Discussion by Dickmonald** [runelite/runelite#17032] [Ideas]
Looking to see if there's a plugin out there that can create an item placeholder for an item you haven't gotten yet. Just looking for more organization in my bank (ironman). I'm not sure if this is even possible, just looking for some insight.

Thanks!

**Comment by Nightfirecat** [comment]
It is not possible to do this.

**Comment by DominickCobb-rs** [comment]
You can have someone give you their bank tags with tagged items you don't have. The bank tags will have placeholders for items you don't have, but otherwise what NFC is the answer.
